pontoon/base/models.py

```diff
@@ -750,7 +750,8 @@ def for_project_locale(self, project, locale, paths=None, search=None):
                 'marked': entity.marked,
                 'original_plural': entity.string_plural,
                 'marked_plural': entity.marked_plural,
-                'key': entity.key,
+                # Remove source string fom the key
+                'key': entity.key.split('\x04')[0],
```

Comment:
We could switch this magic hex into EOT from https://docs.python.org/2/library/curses.ascii.html. It would be good to add little info about why we're doing it and what components requires this particular format. It will be valuable for new contributors.